pom.xml

```diff
@@ -1,12 +1,43 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+
 	<groupId>org.reactome.release.updateDOIs</groupId>
 	<artifactId>update-dois</artifactId>
-	<packaging>jar</packaging>
 	<version>1.0.2-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
 	<name>update-dois</name>
+	<description>Updates DOIs of pathways tagged as needing them</description>
```

Comment:
Would the 'as needing them' phrase be better put as 'Update DOIs of pathways that were updated significantly between releases', or something like that? That way the context of why we need a step for this is task is available

Reply:
Done: https://github.com/reactome/release-update-dois/pull/2/commits/3f84fcda7e4cd812821a2abf4a86d8f1e0e7443a